src/components/editor/Store.tsx

```diff
@@ -54,7 +55,14 @@ const getInitialState = <T,>(
         setSpecs: (newVal) => {
```

Comment:
Whenever possible, I believe function arguments should be descriptive as opposed to generic. Is there a reason why this argument has a generic name?